RetailCoder.VBE/Refactorings/Rename/RenameEventHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Rubberduck.Common;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.UI;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Rubberduck.Refactorings.Rename
+{
+    public class RenameEventHandler : RenameHandlerBase
```

Comment:
Fair enough - I think I'd keep the refactoring tightly coupled with the "specialized rename helpers" then, so that everything is still treated as a single unit.